### PR TITLE
Issues 46/81 Params cleanup

### PR DESCRIPTION
**Pull request by bentsherman**
Issues #46 #81

## Description
Updates some of the publish params and other params to match recent changes to the pipeline script. Also tried to addess some of the issues that were raised in issue #46 but most of those issues have either been dealt with or cannot be helped at the moment.

## Testing
The usual command, you can try different aligners and output formats.
```
nextflow run main.nf -profile testing -with-docker
```

### CODE REVIEW

**Comment by spficklin** [comment]
Hi @bentsherman. I just committed a bunch of documentation updates that @JohnHadish made.  This PR now has a merge conflict.  Can you just double check that your fixes still work for the updated docs?   Thanks.

File: nextflow.config.example [hunk 1]
```diff
@@ -54,6 +56,13 @@ params {
     //
     sra_download = 0
```
Comment (spficklin) [inline nextflow.config.example]: Does sra_download get used for anything?

Comment (bentsherman) [reply nextflow.config.example]: That's from the aspera PR, it allows the user to choose between Aspera or the SRA toolkit to download SRA files.

**Comment by bentsherman** [comment]
@spficklin Just resolved the merge conflict. As long as the CI test passes it should be ready to merge, as John's PR consisted solely of documentation changes anyway.

**Comment by spficklin** [comment]
Thanks for the fixes @bentsherman

**Pull request merged**